src/analytics/Events.tsx

```diff
@@ -601,6 +601,11 @@ export enum AssetsEvents {
   tap_asset = 'tap_asset', // When a user taps on an asset
 }
 
+export enum NftEvents {
+  nft_error_screen_open = 'nft_error_screen_open', // When the high level error screen is mounted
```

Comment:
All screen mounts should already be captured. https://github.com/valora-inc/wallet/blob/69729e8213368db34bde7285eff523be2073d14c/src/navigator/NavigatorWrapper.tsx#L158

But apparently something broken https://linear.app/valora/issue/RET-423/[app-analytics]-global-screen-navigation-events
Would love for us to understand this, instead of manually doing this.

Nothing to change here though.